src/v/utils/to_string.h

```diff
@@ -110,4 +111,20 @@ std::ostream& operator<<(std::ostream& o, const absl::node_hash_map<K, V>& r) {
     o << "}";
     return o;
 }
+
 } // namespace absl
+
+template<typename K, typename V>
+std::ostream& operator<<(std::ostream& o, const chunked_hash_map<K, V>& r) {
```

Comment:
Prefer fmt::formatter<> specializations, if possible: not sure if there was a specific reason we needed ostream<< here?

Reply:
Also prefer "hidden friend" for operator<< when possible though I guess chunked_map is a third party thing so that may not be possible.

Reply:
> if possible: not sure if there was a specific reason we needed ostream<< here?

Seems like some partial specialization nonsense I can't seem to figure out not.